docusaurus/docs/manage/02-router/02-configuration.md

```diff
@@ -0,0 +1,300 @@
+In this example code, the ports are 80 and 443. Set environmental variables to match this type of deployment, and run the command shown. 
+```bash
+ZITI_CTRL_ADVERTISED_ADDRESS=controller01.zitinetwork.example.org
+ZITI_CTRL_PORT=80
+ZITI_EDGE_ROUTER_RAWNAME=router01.zitinetwork.example.org
+ZITI_EDGE_ROUTER_PORT=443
+ROUTER_NAME=$ZITI_EDGE_ROUTER_RAWNAME
+ZITI_ROUTER_IDENTITY_CERT="~/.ziti/config/certs/${ROUTER_NAME}.cert"
+ZITI_ROUTER_IDENTITY_SERVER_CERT="~/.ziti/config/certs/${ROUTER_NAME}.server.chain.cert"
+ZITI_ROUTER_IDENTITY_KEY="~/.ziti/config/certs/${ROUTER_NAME}.key"
+ZITI_ROUTER_IDENTITY_CA="~/.ziti/config/certs/${ROUTER_NAME}.cas"
+
+./ziti create config router edge --routerName  $ROUTER_NAME \
+                                --output $ROUTER_NAME.yaml \
+                                --disableTunneler
+```
+**Expected File Content**
+```yaml
+v: 3
+
+identity:
+    cert:                 "~/.ziti/config/certs/router01.zitinetwork.example.org.cert"
+    server_cert:          "~/.ziti/config/certs/router01.zitinetwork.example.org.server.chain.cert"
+    key:                  "~/.ziti/config/certs/router01.zitinetwork.example.org.key"
+    ca:                   "~/.ziti/config/certs/router01.zitinetwork.example.org.cas"
+
+ctrl:
+    endpoint:             tls:controller01.zitinetwork.example.org:80
+
+link:
+    dialers:
+        - binding: transport
+    listeners:
+        - binding:          transport
+        bind:             tls:0.0.0.0:10080
+        advertise:        tls:router01.zitinetwork.example.org:10080
+        options:
+            outQueueSize:   4
+
+listeners:
+# bindings of edge and tunnel requires an "edge" section below
+    - binding: edge
+    address: tls:0.0.0.0:443
+    options:
+        advertise: router01.zitinetwork.example.org:443
+        connectTimeoutMs: 1000
+        getSessionTimeout: 60
+#  - binding: tunnel
+#    options:
+#
+#
+#
+
+edge:
+    csr:
+        country: US
+        province: NC
+        locality: Charlotte
+        organization: NetFoundry
+        organizationalUnit: Ziti
+        sans:
+            dns:
+                - router01.zitinetwork.example.org
+                - localhost
+            ip:
+                - "127.0.0.1"
+
+#transport:
+#  ws:
+#    writeTimeout: 10
+#    readTimeout: 5
+#    idleTimeout: 5
+#    pongTimeout: 60
+#    pingInterval: 54
+#    handshakeTimeout: 10
+#    readBufferSize: 4096
+#    writeBufferSize: 4096
+#    enableCompression: true
+#    server_cert: ~/.ziti/config/certs/router01.zitinetwork.example.org.server.chain.cert
+#    key: ~/.ziti/config/certs/router01.zitinetwork.example.org.key
+
+forwarder:
+    latencyProbeInterval: 10
+    xgressDialQueueLength: 1000
+    xgressDialWorkerCount: 128
+    linkDialQueueLength: 1000
+    linkDialWorkerCount: 32
+```
+</TabItem>
+<TabItem value="Public-Fabric" label="Public Router" attributes={{className: styles.orange}}>
```

Comment:
Please comment this placeholder so it doesn't show up in the rendering.

Reply:
yes, I created a placeholder for this not to forget. Before this PR merge was completed, I was going to fill this in once ```ziti create config router edge``` changes were merged into the new ziti release.

Reply:
I will remove this for now, we will add the fabric router once the cli option to create transit routers is  added.